### PR TITLE
TR-CERT/CVE-2022-3792/20221219

### PR DESCRIPTION
**Pull request by TR-CERT**

### CODE REVIEW

File: 2022/3xxx/CVE-2022-3792.json [hunk 1]
```diff
@@ -1,18 +1,110 @@
 {
```
Comment (cve-team) [inline 2022/3xxx/CVE-2022-3792.json:1]: File name CVE ID and JSON CVE ID do not match.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/3xxx/CVE-2022-3792.json [hunk 2]
```diff
@@ -1,18 +1,110 @@
...
+        "value": "This issue affects:\nTerminal Operating System versions before 5.0.13"
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://www.usom.gov.tr/bildirim/tr-22-0747-2",
+        "name": ""
```
Comment (anthnysingleton) [inline 2022/3xxx/CVE-2022-3792.json]: If there is no name for the reference, please remove the empty attribute.

File: 2022/3xxx/CVE-2022-3792.json [hunk 3]
```diff
@@ -1,18 +1,110 @@
...
+      },
+      {
+        "refsource": "CONFIRM",
+        "url": "https://omrylmz.com/cve-2022-3792-terminal-operation-system/",
+        "name": ""
```
Comment (anthnysingleton) [inline 2022/3xxx/CVE-2022-3792.json]: If there is no name for the reference, please remove the empty attribute.

File: 2022/3xxx/CVE-2022-3792.json [hunk 4]
```diff
@@ -1,18 +1,110 @@
...
+      },
+      {
+        "refsource": "CONFIRM",
+        "url": "https://fordefence.com/cve-2022-3792-gullseye-terminal-operation-system/",
+        "name": ""
```
Comment (anthnysingleton) [inline 2022/3xxx/CVE-2022-3792.json]: If there is no name for the reference, please remove the empty attribute.

**Review by anthnysingleton — COMMENTED** [review]
Hello @TR-CERT ,

This PR is held up because the reference object contains attributes called "name" that are set to empty strings. If you can update the file and remove those attributes we can then process the PR.

Thanks,
CVE Content Team

**Pull request merged**